core/templates/components/forms/slider/audio-slider.component.ts

```diff
@@ -40,8 +40,9 @@ import { downgradeComponent } from '@angular/upgrade/static';
   }`]
 })
 export class AudioSliderComponent {
-  @Input() value: number;
-  @Input() max: number;
+  // Refer TS-7-1 from the wiki.
```

Comment:
Will need to update this with the section link when the wiki is up, and also add a one liner description as we discussed before.

Reply:
done